src/uu/factor/src/numeric/mod.rs

```diff
@@ -0,0 +1,18 @@
+// * This file is part of the uutils coreutils package.
+// *
+// * (c) 2020 nicoo            <nicoo@debian.org>
+// *
+// * For the full copyright and license information, please view the LICENSE file
+// * that was distributed with this source code.
+
+mod gcd;
+pub use gcd::gcd;
+
+mod traits;
+use traits::{DoubleInt, Int, OverflowingAdd};
```

Comment:
could you please fix this warning ?

Reply:
Done

Reply:
Reverted the “fix”, as the warning is erroneous.  *sigh*

Reply:
Fixed for good now.